templates/web/base/admin/extra-metadata-form.html

```diff
@@ -0,0 +1,81 @@
+<ul class="js-metadata-items">
+  [% FOR meta IN metas.merge([{}]) %]
+    <li class="js-metadata-item [% IF loop.last %]js-hidden js-metadata-item-template[% END %]" data-index="[% loop.index %]">
+        <button class="btn btn--small js-metadata-item-remove js-visible">[% loc('Remove field') %]</button>
+
+        <div class="admin-hint"><p>[% loc('The ordering of this field on the report page. Fields are shown in ascending order according to this value.') %]</p></div>
+        <label>
+            [% loc('Order') %]
+            <input name="metadata[[% loop.index %]].order" data-field-name="order" type=text value="[% meta.order | html %]">
+        </label>
+
+        <div class="admin-hint"><p>[% loc('The code used to store this field value in the database. e.g. <code>address</code> would be available as <code>problem.extra.address</code> in the templates.') %]</p></div>
+        <label>
+            [% loc('Code') %]
+            <input name="metadata[[% loop.index %]].code" data-field-name="code" type=text value="[% meta.code | html %]">
+        </label>
+
+        <div class="admin-hint"><p>[% loc('Whether the user is required to provide a value for this field.') %]</p></div>
+        <label>
+            [% loc('Required') %]
+            <input name="metadata[[% loop.index %]].required" data-field-name="required"  type=checkbox [% meta.required == 'true' ? 'checked' : '' %]>
+        </label>
+
+        <div class="admin-hint"><p>[% loc('If unticked the user won’t see an input field, just the ‘Description’ text. This should generally be ticked.') %]</p></div>
+        <label>
+            [% loc('Variable') %]
+            <input name="metadata[[% loop.index %]].variable" data-field-name="variable"  type=checkbox [% meta.variable == 'true' ? 'checked' : '' %]>
+        </label>
+
+        <div class="admin-hint"><p>[% loc('The field name as shown to the user on the report form.') %]</p></div>
+        <label>
+            [% loc('Description') %]
+            <input name="metadata[[% loop.index %]].description" data-field-name="description" type=text value="[% meta.description | html %]">
+        </label>
+
+        <div class="admin-hint"><p>[% loc('Some extra guidance text shown to the user alongside the field. The default template does not show this (<code>meta.datatype_description</code>), you must add it in <code>category_extras_fields.html</code>') %]</p></div>
```

Comment:
It sounds like datatype_description is for describing the type of data - e.g. it's date input or something? Anyway, as you say, not used so guess people can use it whatever they want. Open311s we've seen do put things like "A string" in this field.